### ISSUE TITLE
Port Rayleigh Friction to CCPP

### ISSUE THREAD

**Issue by andrewgettelman** [ESCOMP/CAM#431]
Following the guide,  I'm going to try porting Rayleigh Friction (all 8 lines of it) to the CCPP....

**Comment by dan800** [comment]
Andrew, is the the guide you are using? 
[https://docs.google.com/document/d/1oPgEhhZkqAuyv_SPbGP299Qk2MWD-5lw4zSBqT2XG08/edit?usp=sharing](url)

**Comment by andrewgettelman** [comment]
Yes @dan800 that is it.  Waiting now on confirmation that CCPP adjustments for tendency updates are done, then I can finish it off.

**Comment by jimmielin** [comment]
Resolved by https://github.com/ESCOMP/CAM/pull/1245 cam6_4_086
